App/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,109 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+    // switch to true to log data before validating 
+    private static final boolean DEBUGGING = true;
+
+    private void serverLog(
+        String name, String gender, Integer age, Boolean medicalAssistance, Integer numberOfFamilyMembers, 
+        Boolean disabilities, Integer days
+    ) {
+        System.out.println("------------------------");
+        System.out.println("Name: " + name);
+        System.out.println("Age: " + age);
+        System.out.println("Gender: " + gender);
+        System.out.println("# Family members: " + numberOfFamilyMembers);
+        System.out.println("Disabilities: " + disabilities);
+        System.out.println("Medical Assistance: " + medicalAssistance);
+        System.out.println("Days: " + days);
+        System.out.println("------------------------");
+    }
+
+    private boolean checkIfValid(
+        String name, String gender, Integer age, Boolean medicalAssistance, Integer numberOfFamilyMembers, 
```

Comment:
Same comment here as above, minus the toString thing - you'll definitely want to change this to just accept a User once that exists.

If you wanted to get fancy, you could add this as an isValid() method to the User class, or even (maybe?) raise an exception in the User constructor if one of these is wrong.

Reply:
100% agree. I feel like a lot if not all of these methods would be much better suited to User, I would definitely be up to take on that fanciness